src/cosmology/api/__init__.py

```diff
@@ -1,19 +1,54 @@
 """The Cosmology API standard."""
 
 from cosmology.api._components import (
-    HasBaryonComponent,
-    HasDarkEnergyComponent,
-    HasDarkMatterComponent,
-    HasGlobalCurvatureComponent,
-    HasMatterComponent,
-    HasNeutrinoComponent,
-    HasPhotonComponent,
-    HasTotalComponent,
+    BaryonComponent,
+    CurvatureComponent,
+    DarkEnergyComponent,
+    DarkMatterComponent,
+    HasMNu,
+    HasNeff,
+    HasOmegaB,
+    HasOmegaB0,
+    HasOmegaDE,
+    HasOmegaDE0,
+    HasOmegaDM,
+    HasOmegaDM0,
```

Comment:
Oh no. It's not impossible, but my inclination is that we would have some namespace depth if forced to do that.
In our discussions we thought that `omega_m` would be an "extension" to the API, e.g. bundled with a cosmic-shear focused set of Protocols, so perhaps it would be
`api.shear.HasOmegaM` for `omega_m`.